src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -62,23 +68,57 @@ public QAReport executeQACheck() throws Exception
      * @return boolean -- True if a new, unreviewed regulation instance has been added.
      * @throws Exception -- DBA exceptions.
      */
-    private boolean newRegulationWithoutReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+    private boolean changedRegulatedByWithoutNewReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+
         // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
-        if (previousRlE != null) {
-            int currentRlERegulationCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
-            int currentRlEReviewedCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
-            int previousRlERegulationCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
-            int previousRlEReviewedCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+        if (previousRlE == null) {
+            return false;
+        } else {
```

Comment:
Since the if condition is a guard and returns from the method, the else can actually be removed because the method execution won't go any farther when it `previousRlE` is null.